src/main/java/games/strategy/engine/data/TripleaHandler.java

```diff
@@ -0,0 +1,393 @@
+package games.strategy.engine.data;
+
+import static tools.map.xml.creator.MapXmlHelper.XML_NODE_NAME_ALLIANCE;
```

Comment:
There probably should be no dependency from `games.strategy.*` to `tools.*`.  Vice versa seems to be acceptable, as that dependency path is already present in multiple locations within the tool code.